.github/workflows/e2e-unmanaged-cluster.yaml

```diff
@@ -0,0 +1,57 @@
+---
+name: E2E Test - Unamanged Cluster
+
+on:
+  repository_dispatch:
+    types: [daily-build]
+
+  pull_request_target:
+    types: [opened, reopened, synchronize, ready_for_review]
+    paths:
+      - cli/cmd/plugin/unmanaged-cluster
+
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+      - release-*
+
+    paths:
+      - "cli/cmd/plugin/unmanaged-cluster/*.go"
+      - "cli/cmd/plugin/"
+      - "**.sh"
```

Comment:
Do we want to run the unmanaged-cluster e2e tests on changes to _any and all_ scripts in this project?

For example, this means we will run these e2e tests on changes to the `hack/dailybuild/publish-daily-build.sh` script which doesn't have anything to do with `unmanaged-clusters` or these tests directly.

Reply:
Agree but I thought if there is any changes in `build-tce.sh` or `download-or-build-tce.sh` then we can test unmanaged cluster creation is healthy or not but yeah we can look in daily build.